src/pages/dapps.tsx

```diff
@@ -126,252 +116,154 @@ import wallet from "@/public/images/wallet.png" // width=300
-    lineHeight="22px"
-    letterSpacing={0}
-    mt={2}
-    {...props}
-  />
+  <h2 className="mb-8 mt-2 text-2xl" {...props} />
 )
 
-const H2 = (props: HeadingProps) => (
-  <Heading
-    mt={12}
-    mb={8}
-    fontSize={{ base: "2xl", md: "2rem" }}
-    fontWeight="semibold"
-    lineHeight={1.4}
+const H2 = (props: BaseHTMLAttributes<HTMLHeadingElement>) => (
+  <h2
+    className={cn(
+      "mb-8 mt-12 text-2xl font-semibold md:text-3xl",
+      props.className
+    )}
     {...props}
   />
 )
 
-const H3 = (props: HeadingProps) => (
-  <OldHeading
-    as="h3"
-    fontSize={{ base: "xl", md: "2xl" }}
-    fontWeight="semibold"
-    lineHeight={1.4}
-    {...props}
-  />
+const H3 = (props: BaseHTMLAttributes<HTMLHeadingElement>) => (
+  <h3 className="mb-4 mt-8 text-xl font-semibold md:text-2xl" {...props} />
 )
 
 const StyledH3 = (props: ChildOnlyProp) => (
-  <Heading
-    as="h3"
-    lineHeight={1.4}
-    fontSize="xl"
-    fontWeight="bold"
-    mb={2}
-    mt={6}
-    sx={{
-      a: {
-        dispalay: "none",
-      },
-    }}
-    {...props}
-  />
+  <h3 className="mb-2 mt-6 text-xl font-bold" {...props} />
 )
 
 const StyledInfoBanner = (props: ComponentPropsWithRef<typeof InfoBanner>) => (
   <InfoBanner w={{ lg: "50%" }} {...props} />
 )
 
 const Column = (props: ChildOnlyProp) => (
-  <Box flex="1 1 75%" mb={6} me={{ lg: 8 }} {...props} />
+  <div className="mb-6 flex-[1_1_75%] lg:me-8" {...props} />
 )
 
 const FullWidthContainer = (
   props: ChildOnlyProp & { ref: React.RefObject<HTMLDivElement> }
 ) => (
   <Page
-    m={0}
-    mb={16}
-    pt={16}
-    pb={8}
-    borderTop="1px solid"
-    borderColor="border"
-    bg="ednBackground"
+    className="m-0 mb-16 border-t border-border bg-background-highlight pb-8 pt-16"
     {...props}
   />
 )
 
 const CardContainer = (props: ChildOnlyProp) => (
-  <SimpleGrid gap={4} columns={[1, null, 2]} {...props} />
+  <div className="grid grid-cols-1 gap-4 md:grid-cols-2" {...props} />
 )
 
 const StepBoxContainer = (props: ChildOnlyProp) => (
   <Flex
-    flexWrap={{ base: "wrap", lg: "nowrap" }}
-    w="full"
-    my={4}
-    mb={16}
-    mx={0}
+    className="mx-0 my-4 mb-16 w-full flex-wrap lg:flex-nowrap"
     {...props}
   />
 )
 
 const StepBox = (props: ComponentPropsWithRef<typeof BaseLink>) => (
   <BaseLink
-    border="1px solid"
-    borderColor="border"
-    pt={0}
-    pb={{ base: 8, md: 0 }}
-    px={8}
-    display="flex"
-    flexDirection={{ base: "column", md: "row" }}
-    justifyContent="space-between"
-    alignItems={{ base: "flex-start", md: "center" }}
-    color="text"
-    textDecor="none"
-    w="full"
-    transition="transform 0.2s"
-    _hover={{
-      bg: "ednBackground",
-      transform: "scale(1.05)",
-    }}
+    className="flex w-full flex-col items-start justify-between border border-border px-8 pb-8 pt-0 no-underline transition-transform duration-200 hover:scale-105 hover:bg-background md:flex-row md:items-center md:pb-0"
```

Comment:
We need to add `text-body` to override the primary color. This is how it looks like atm
![image](https://github.com/user-attachments/assets/27fdf575-d61d-4865-9439-9e428cf57461)

and how it should look like
![image](https://github.com/user-attachments/assets/b23853a4-b158-4785-861a-517ca6e9efeb)

```suggestion
    className="flex w-full flex-col items-start justify-between border border-border px-8 pb-8 pt-0 no-underline transition-transform duration-200 hover:scale-105 hover:bg-background-highlight md:flex-row md:items-center md:pb-0"
```

Reply:
That's noted. Fix coming soon